### PR TITLE
Fix icon sprite and optimize capabilities

### PR DESCRIPTION
**Pull request by udhadpa**
Hi,
This PR includes
1. Fix icon sprite script by removing it from the head tag to the body tag for all the fixture files.
2. Optimizing code for setting capabilities in Base class.
@umahaea for your review. Please let me know your feedback
Thanks,

### CODE REVIEW

File: src/main/java/utilities/BaseClass.java [hunk 1]
```diff
@@ -111,31 +116,22 @@ protected void setUp() throws MalformedURLException {
...
                 caps.setCapability(CapabilityType.LOGGING_PREFS, logs);
-                caps.setCapability("platform", platform);
-                caps.setCapability("version", sauceBrowserVer);
-                caps.setCapability("maxDuration", "10800");
-                caps.setCapability("name", this.getClass().getSimpleName());
-                if (platform.startsWith("Windows")) {
-                    caps.setCapability("screenResolution", "2560x1600");
-                } else if (platform.startsWith("OS X")) {
-                    caps.setCapability("screenResolution", "2048x1536");
+                for (int i = 0; i < (desktopCaps.length - 1); i = i + 2) {
```
Comment (mhomolak) [inline src/main/java/utilities/BaseClass.java]: All of the i=i+2 can be changed to i+=2

119, 132, 158

Comment (udhadpa) [inline src/main/java/utilities/BaseClass.java]: Oh yes, din't think of that. Thanks Mike 👍

File: src/main/java/utilities/BaseClass.java [hunk 2]
```diff
@@ -86,16 +88,22 @@
...
     @BeforeClass(alwaysRun = true)
     protected void setUp() throws MalformedURLException {
         caps = new DesiredCapabilities();
         setDesktop = desktop;
         setMobile = mobile;
         logs.enable(LogType.BROWSER, Level.ALL);
+        String[] desktopCaps = new String[]{"platform", platform, "version", sauceBrowserVer, "maxDuration", "10800", "name", this.getClass().getSimpleName(), "tunnel-identifier", System.getenv("TRAVIS_JOB_NUMBER"), "build", System.getenv("TRAVIS_BUILD_NUMBER"), "screenResolution", "2048x1536"};
+        String[] mobileCaps = new String[]{MobileCapabilityType.DEVICE_NAME, mobDeviceName, MobileCapabilityType.PLATFORM_VERSION, mobilePlatformVer, MobileCapabilityType.BROWSER_NAME, mobBrowser, MobileCapabilityType.APPIUM_VERSION, appiumVer, "maxDuration", "10800", "name", this.getClass().getSimpleName(), "tunnel-identifier", System.getenv("TRAVIS_JOB_NUMBER"), "build", System.getenv("TRAVIS_BUILD_NUMBER")};
 
+        if (!((desktopCaps.length % 2 == 0) && (mobileCaps.length % 2 == 0))) {
```
Comment (umahaea) [inline src/main/java/utilities/BaseClass.java]: Love this conditional check!

Comment (udhadpa) [inline src/main/java/utilities/BaseClass.java]: learnt from the best ;)

**Comment by udhadpa** [comment]
Got enough approvals, good to merge

File: src/main/java/compounds/fixtures/alerts.html [hunk 3]
```diff
@@ -6,25 +6,25 @@
...
-    </script>
 </head>
 
 <body>
 <h2>Compounds - Banner Style Alerts</h2>
 
+<script>
+    if (!document.getElementById('pe-icons-sprite')) {
+        var pe_ajax = new XMLHttpRequest();
+        pe_ajax.open("GET", "../../../../../icons/p-icons-sprite-1.1.svg", true);
```
Comment (JasonLantz) [inline src/main/java/compounds/fixtures/alerts.html]: this icons path looks different from the rest...

Comment (JasonLantz) [inline src/main/java/compounds/fixtures/alerts.html]: Mike says this is cool..... disregard...

**Pull request merged**